### PR TITLE
Fix #1685 change fileName attributes to filename

### LINKED ISSUE #1685: Inconsistent attribute "filename" across different pipes
**Describe the issue**
Whilst replacing the deprecated FilePipe I had to replace it with a FixedResultPipe and ran into the fact that the attribute "fileName" has different casing as the LocalFileSystemPipe's attribute "filename".

FixedResultPipe:
image

LocalFileSystemPipe:
image

**Reporter**
Laurens Mäkel

**To Reproduce**

**Screenshots**

**Configuration**
```
Please provide the configuration of the Pipe or Receiver with the problem.
```

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```

**Environment**
```
Please go to the console of your Frank, click on 'Information' and copy-paste the results here. 
```

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql,PostgreSQL] Not relevant
 - Browser: [e.g. Chrome, Safari, Edge, Firefox] Not relevant

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

### PR DESCRIPTION
**Pull request by alisihab**

### CODE REVIEW

File: core/src/main/java/nl/nn/adapterframework/errormessageformatters/FixedErrorMessage.java [hunk 1]
```diff
@@ -94,16 +95,20 @@ public String getReturnString() {
 		return returnString;
 	}
 
+	@Deprecated
+	@ConfigurationWarning("attribute 'fileName' is replaced with 'filename'")
+	public void setFileName(String fileName) {
+		this.filename = fileName;
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/errormessageformatters/FixedErrorMessage.java:101]: it's better to call the setter :)

File: core/src/main/java/nl/nn/adapterframework/jdbc/ResultSet2FileSender.java [hunk 2]
```diff
@@ -143,12 +144,18 @@ public String getStatusFieldType() {
 		return statusFieldType;
 	}
 
-	@IbisDoc({"the session key that contains the name of the file to use", ""})
+	@Deprecated
+	@ConfigurationWarning("attribute 'fileNameSessionKey' is replaced with 'filenameSessionKey'")
 	public void setFileNameSessionKey(String filenameSessionKey) {
-		this.fileNameSessionKey = filenameSessionKey;
+		this.filenameSessionKey = filenameSessionKey;
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/jdbc/ResultSet2FileSender.java:150]: its better to call the new setter

File: core/src/main/java/nl/nn/adapterframework/pipes/FilePipe.java [hunk 3]
```diff
@@ -101,12 +101,18 @@ public void setWriteSuffix(String suffix) {
...
+		setFilename(filename);
 	}
 
-	public void setFileNameSessionKey(String filenameSessionKey) {
-		fileHandler.setFileNameSessionKey(filenameSessionKey);
+	public void setFilename(String filename) {
+		fileHandler.setFilename(filename);
+	}
+
+	public void setFilenameSessionKey(String filenameSessionKey) {
```
Comment (gvanbrakel) [inline core/src/main/java/nl/nn/adapterframework/pipes/FilePipe.java:114]: You should keep the old setter too, like setFileName()

File: core/src/main/java/nl/nn/adapterframework/pipes/MessageSendingPipe.java [hunk 4]
```diff
@@ -1339,12 +1340,18 @@ public boolean isStreamResultToServlet() {
...
 	public void setStubFileName(String fileName) {
-		stubFileName = fileName;
+		setStubFilename(fileName);
+	}
+
+	@IbisDoc({"25", "when set, the pipe returns a message from a file, instead of doing the regular process", ""})
+	public void setStubFilename(String filename) {
+		stubFilename = filename;
 	}
 	public String getStubFileName() {
```
Comment (gvanbrakel) [inline core/src/main/java/nl/nn/adapterframework/pipes/MessageSendingPipe.java:1353]: Rename this one too, or replace with ``@Getter``

**Review by gvanbrakel — CHANGES_REQUESTED** [review]
It would also be good to make a note of this change in RELEASES.MD

**Comment by alisihab** [comment]
> It would also be good to make a note of this change in RELEASES.MD

I am not sure about it since we show configuration warning(deprecation notice) in the console.

**Review by gvanbrakel — APPROVED** [review]

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**